src/Components/Cards/cards.css

```diff
@@ -92,15 +92,42 @@
   display: flex;
   flex-direction: column;
   cursor: pointer;
+  position: relative;
 }
 
-.videolisting-content{
-  font-size: 1.6rem ;
+.videolisting-content {
+  font-size: 1.6rem;
   padding-top: 1rem;
   display: flex;
 }
 
-.kebab-menu{
+.kebab-menu {
   display: flex;
   align-items: center;
-}
\ No newline at end of file
+}
+
+.modal {
+  font-size: 1.6rem;
+  position: absolute;
+  right: 0rem;
+  top: 105%;
+  z-index: 2;
+  color: white;
+  width: 26rem;
+  height: max-content;
+  background-color: #202020;
```

Comment:
if this background color is being used multiple times then you should make it a variable